src/components/join/Info.jsx

```diff
@@ -1,7 +1,20 @@
+import InfoCard from "./InfoCard";
+
 const Info = () => {
   return (
-    <div className="">
-      <p>Info</p>
+    <div className="w-full l-100 flex">
+      <div className="flex flex-col ml-28 mt-24">
+        <p className="text-swim-yellow md:text-2xl">Lorem ipsum dolor sit</p>
+        <p className="text-swim-blue-400 md:text-4xl ">
+          How to keep up with <br />
+          information, <br />
+          competitions, and <br />
+          socials:
+        </p>
+      </div>
+      <div className="ml-auto">
+        <InfoCard />
```

Comment:
<img width="417" alt="322137650-f057a122-6e52-4c69-a88b-96bc48751b28" src="https://github.com/acm-ucr/ucr-swim-website/assets/91030482/5c66d73e-8abd-4251-938d-f5ab9ed8b19b">

add the button at the bottom. there is a component created for you called Button.jsx. correct the spelling on form.